eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -205,20 +205,30 @@ jobs:
       displayName: Export VMR sources
       workingDirectory: $(Build.StagingDirectory)
 
-  - ${{ if ne(parameters.reuseBuildArtifactsFrom, '') }}:
-    - download: current
-      artifact: ${{ parameters.reuseBuildArtifactsFrom }}_${{ parameters.architecture }}_Artifacts
-      patterns: |
-        **/Private.SourceBuilt.Artifacts.*.tar.gz
-        **/dotnet-sdk-*.tar.gz
-      displayName: Download Previous Build
-
-    - task: CopyFiles@2
-      displayName: Copy Previous Build
-      inputs:
-        SourceFolder: $(Pipeline.Workspace)/${{ parameters.reuseBuildArtifactsFrom }}_${{ parameters.architecture }}_Artifacts/assets/Release
-        Contents: '*.tar.gz'
-        TargetFolder: $(sourcesPath)/prereqs/packages/archive/
+  - ${{ if ne(parameters.reuseBuildArtifactsFrom,'') }}:
+    - ${{ each reuseBuildArtifacts in parameters.reuseBuildArtifactsFrom }}:
+      - download: current
+        artifact: ${{ reuseBuildArtifacts }}_Artifacts
+        patterns: |
+          **/Private.SourceBuilt.Artifacts.*.tar.gz
```

Comment:
Do any of these get downloaded and used? My gut is telling me that after the publishing changes that @NikolaMilosavljevic did we would not use these any longer in MS VMR builds.

Reply:
Uh... they're used by _source build_ but not _VMR_, for where those divides still exist. I validated that I didn't break that stuff in https://dev.azure.com/dnceng-public/public/_build/results?buildId=803686&view=logs&j=3121c41d-6088-53a5-4ef6-9a86f3460fe9&t=2a1e1853-73d8-57b7-044a-d1b65dc9fcea (yes that's a failure, but it's the same failure currently in main, rather than a new failure introduced by my changes)